docs/RELEASE_POLICY.md

```diff
@@ -86,8 +86,9 @@ also be specifying the supported kernel versions.
 
 | Release | Release Date | Latest Patch | Min. end of support | Official end of Support        |
 | ------: | -----------: | -----------: | ------------------: | :----------------------------- |
+| v1.3    |   2023-03-02 | v1.3.0       |          2023-09-02 | Supported                      |
```

Comment:
nit: latest patch should be 1.3.1, no?

Reply:
My understanding that this is done after the release is done. @pb8o am I right?

Reply:
Yeah we can do it after.. I think this change may only make sense in `main` anyway...